.circleci/config.yml

```diff
@@ -179,6 +179,11 @@ jobs:
     steps:
       - checkout
       - install-dependencies
+      - install-and-create-sim:
+          install-name: iOS 15.4 Simulator
+          sim-device-type: iPhone-13
+          sim-device-runtime: iOS-15-4
+          sim-name: iPhone 13 (15.4)
```

Comment:
We still need to wait for 15.5 though.

Reply:
Yup, without digging into it, I thought we might have it available because it's on my machine with the same version of Xcode as CircleCI.